src/Silverback.Integration/Messaging/Messages/DefaultMessageHeaders.cs

```diff
@@ -99,5 +99,23 @@ public static class DefaultMessageHeaders
         ///     consumer side to determine the correct key to be used to decrypt the message.
         /// </remarks>
         public const string EncryptionKeyId = "x-encryption-key-id";
+
+        /// <summary>
+        ///     This will be set by the Move error policy and will contain the reason the failed
+        ///     message is being moved.
+        /// </summary>
+        public const string FailureReason = "x-failure-reason";
+
+        /// <summary>
+        ///     This will be set by the Move error policy and will contain the timestamp of the failed
+        ///     message.
+        /// </summary>
+        public const string SourceTimestamp = "x-source-timestamp";
```

Comment:
This is also Kafka specific, isn't it? Shouldn't we move it together with topic/offset etc.?

Reply:
Right, moved to `KafkaMessageHeaders`.